api/src/main/java/io/minio/messages/QueueConfiguration.java

```diff
@@ -97,14 +97,36 @@ public void setEvents(List<EventType> events) {
   /**
    * Returns filter.
    */
+  public Filter getFilter() {
+    return filter;
+  }
+
+
+  /**
+   * Sets filter.
+   */
+  public void setFilter(Filter filter) {
+    this.filter = filter;
+  }
+
+  /**
+   * Returns filter..
+   *
+   * @see #getFilter
+   * @deprecated replaced by getFilter()
+   */
+  @Deprecated
   public Filter filter() {
     return filter;
   }
 
 
   /**
    * Sets filter.
+   *  @see #setFilter
+   *  @deprecated replaced by setFilter()
    */
+  @Deprecated
   public void filter(Filter filter) {
```

Comment:
This method needs to be removed because of version bump.

Reply:
IMO, we should keep this method and next version bump we can remove it. We should support both the methods for some time. Please confirm .

Reply:
My recommendation is to remove it now.  There is no need to deprecate this ugly method.

Reply:
Removed.

Reply:
removed.